cogs/funny.py

```diff
@@ -579,6 +583,105 @@ def summer(number_list: list, author: str) -> str:
             nice_print = nice_print[1:]
         return f"{author} rolled **{total}**. ({nice_print})"
 
+    @commands.command()
+    async def joke(self, ctx: Context) -> None:
+        """Send a random joke."""
+        async with self.session.get(
+            "https://mrwinson.me/api/jokes/random"
+        ) as resp:
+            if resp.status == 200:
+                data = await resp.json()
+                joke = data["joke"]
+                embed = Embed(
+                    description=joke,
+                    color=Color.gold()
+                )
+                await ctx.send(embed=embed)
+            else:
+                await ctx.send(f"Something went boom! :( [CODE: {resp.status}]")
+
+    @commands.command()
+    async def koala(self, ctx: Context) -> None:
+        """Get a random picture of a koala."""
+        async with self.session.get(
+            "https://some-random-api.ml/img/koala"
+        ) as resp:
+            if resp.status == 200:
+                data = await resp.json()
+                embed = Embed(
+                    title="Random Koala!",
+                    color=Color.gold()
+                )
+                embed.set_image(url=data["link"])
+                await ctx.send(embed=embed)
+            else:
+                await ctx.send(f"Something went boom! :( [CODE: {resp.status}]")
+
+    @commands.command()
+    async def panda(self, ctx: Context) -> None:
+        """Get a random picture of a panda."""
+        async with self.session.get(
+            "https://some-random-api.ml/img/panda"
+        ) as resp:
+            if resp.status == 200:
+                data = await resp.json()
+                embed = Embed(
+                    title="Random Panda!",
+                    color=Color.gold(),
+                )
+                embed.set_image(url=data["link"])
+                await ctx.send(embed=embed)
+            else:
+                await ctx.send(f"Something went boom! :( [CODE: {resp.status}]")
+
+    @commands.command()
+    async def catfact(self, ctx: Context) -> None:
+        """Send a random cat fact."""
+        async with aiohttp.ClientSession() as session:
+            async with session.get(
+                "https://cat-fact.herokuapp.com/facts"
+            ) as response:
+                self.all_facts = await response.json()
+
+        fact = choice(self.all_facts["all"])
+        await ctx.send(embed=Embed(
+            title="Did you Know?",
+            description=fact["text"],
+            color=0x690E8
+        ))
```

Comment:
https://github.com/Faholan/All-Hail-Chaos/blob/master/cogs/animals.py#L72
If you want, you can edit that command for better formatting